stat/distuv/pareto.go

```diff
@@ -0,0 +1,107 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package distuv
+
+import (
+	"math"
+
+	"golang.org/x/exp/rand"
+)
+
+// Pareto implements the Pareto (Type I) distribution, a one parameter distribution
+// with support above the scale parameter.
+//
+// The density function is given by
+// \frac{\alpha x_m^{\alpha}}{x^{\alpha+1}} for x >= x_m
```

Comment:
We use something that is almost but not quite TeX for these formulae. In this case I'd not use `\frac` but rather a `/` and parens. Also, indent the formula by one space and finish with a dot.